src/main/java/com/minelittlepony/api/pony/PonyPosture.java

```diff
@@ -91,7 +89,8 @@ public static boolean hasSeaponyForm(LivingEntity entity) {
         }
         return Pony.getManager().getPony(entity).filter(pony -> {
             return (pony.race() == Race.SEAPONY
-                    || (entity instanceof PlayerEntity player && SkinsProxy.getInstance().getSkin(DefaultPonySkinHelper.SEAPONY_SKIN_TYPE_ID, player).isPresent())
+                    || (entity instanceof PlayerEntity player &&
+                    SkinsProxy.getInstance().getSkin(DefaultPonySkinHelper.SEAPONY_SKIN_TYPE_ID, player).isPresent())
```

Comment:
There's a lot of these changes that seem to be coming from your IDE attempting to reformat the code. Can you exclude these or try to minimise the actual changes to the code please?